bus/bus.go

```diff
@@ -1531,6 +1544,7 @@ func (b *bus) Handler() http.Handler {
 		"POST   /slabs/refreshhealth": b.slabsRefreshHealthHandlerPOST,
 		"GET    /slab/:key":           b.slabHandlerGET,
 		"PUT    /slab":                b.slabHandlerPUT,
+		"GET    /slabobjects/:key":    b.slabObjectsHandlerGET,
```

Comment:
I don't think this plays well with the other routes we have. I know `/objects/*path` is a little annoying because of the `*` but couldn't we do `/search/objects/slab/:key` or `/search/objects/:slabkey` instead? I noticed `search/objects` has a query string parameter called `key` which searches on `object_id` so we should probably straighten that out also. 

There's no such thing as a `slabobject` and it's kind of weird to return `ObjectMetadata`s from the `slab` endpoints I think.

Reply:
edit: `slab/:key/objects` is what we want I think